### PR TITLE
Update html_to_markdown.rst

### PR DESCRIPTION
**Pull request by ThomasLandauer**
Integrating some findings of https://github.com/twigphp/Twig/issues/3357

### CODE REVIEW

File: doc/filters/html_to_markdown.rst [hunk 1]
```diff
@@ -64,3 +63,23 @@ You can also use the filter on an included file:
...
+
+.. code-block:: twig
+
+    {% apply html_to_markdown({hard_break: false}) %}
+        <html>
+            <h1>Hello!</h1>
+        </html>
+    {% endapply %}
+
+.. note::
```
Comment (stof) [inline doc/filters/html_to_markdown.rst]: why an empty note here ?

Comment (ThomasLandauer) [reply doc/filters/html_to_markdown.rst]: Sorry - copy-paste leftover...  
Deleted now.

Comment (ThomasLandauer) [reply doc/filters/html_to_markdown.rst]: GitHub hangs - I hope it got through...

**Comment by fabpot** [comment]
Thank you @ThomasLandauer.

**Pull request merged**